ui/app/AppLayouts/Chat/ChatColumn/ChatInput.qml

```diff
@@ -58,9 +58,12 @@ Rectangle {
             anchors.right: chatSendBtn.left
             anchors.rightMargin: 16
             anchors.top: parent.top
-            anchors.topMargin: 24
+            anchors.topMargin: 0
+            anchors.bottom: parent.bottom
+            anchors.bottomMargin: 0
             anchors.left: parent.left
             anchors.leftMargin: 24
```

Comment:
Maybe you can use the `leftPadding` and `rightPadding` like you did in the Input?